scipy_jacobian.py

```diff
@@ -0,0 +1,151 @@
+"""
+This is a modified and JIT-ted version of the internal numerical Jacobian approximation used by scipy.integrate.solve_ivp, when no analytical Jacobian is provided.
+The original code can be found at `scipy/integrate/_ivp/common.py` in the SciPy source code.
+"""
+
+import numpy as np
+from numba import njit
+from numba.types import Tuple, float64, int64   # JIT types
+
+ENABLE_JIT = True
```

Comment:
You may disable JIT-ting the jacobian approximator to improve import time by 5-10 sec, however it makes param studies somewhat slower: `ENABLE_JIT = False`

You may also use the old jacobian by using `use_builtin_jac = True` in solve_ivp(). Or provide analytic jacobian by `jac = MyJacFun`